app/src/main/java/com/example/lexis/utilities/Utils.java

```diff
@@ -114,6 +116,35 @@ public static String getCurrentTargetLanguage() {
         return "";
     }
 
+    /*
+    Return the studied languages of the logged-in user.
+    */
+    public static List<String> getCurrentStudiedLanguages() {
+        String objectId = ParseUser.getCurrentUser().getObjectId();
```

Comment:
This could be a good place to null check ^

Reply:
Just to confirm, we'd want to check whether the result of `getCurrentUser()` is null, right? I assumed that a null object ID would result in a ParseException that we can catch, but I see how trying to get the object ID of a null user would cause issues.